anhalyze/core/anhalyze_plot_utils.py

```diff
@@ -44,22 +40,91 @@ def get_feature_mask(feature='land', resolution='50m'):
+            central_longitude=proj_info['central_longitude'],
+            central_latitude=proj_info['central_latitude']),
+        'AlbersEqualArea': ccrs.AlbersEqualArea(
+            central_longitude=proj_info['central_longitude'],
+            central_latitude=proj_info['central_latitude'],
+            standard_parallels=proj_info['standard_parallels']),
+        'NorthPolarStereo': ccrs.NorthPolarStereo(central_longitude=proj_info['central_longitude']),
+        'Orthographic': ccrs.Orthographic(central_longitude=proj_info['central_longitude'],
+                                          central_latitude=proj_info['central_latitude']),
+        'Robinson': ccrs.Robinson(central_longitude=proj_info['central_longitude']),
+        'LambertConformal': ccrs.LambertConformal(central_longitude=proj_info['central_longitude'],
+                                                  standard_parallels=proj_info['standard_parallels']),
+        'Mercartor': ccrs.Mercator(central_longitude=proj_info['central_longitude'],
+                                   min_latitude=proj_info['lat_range'][0],
+                                   max_latitude=proj_info['lat_range'][1]),
+        'AzimuthalEquidistant': ccrs.AzimuthalEquidistant(central_longitude=proj_info['central_longitude'],
+                                                          central_latitude=proj_info['central_latitude']),
+    }
+
+    proj_config = proj_list[proj]
+
+    assert proj in list(proj_list.keys()), \
+        f'[anhalyze_plot_utils] Projection {proj} not found in list of projections available: {list(proj_list.keys())}'
+
+    return proj_config
+
+
+def get_projection_info(data):
+    # Setting up user's region
+    east = data.attrs['coord_lon_range'][1]
+    west = data.attrs['coord_lon_range'][0]
+    north = data.attrs['coord_lat_range'][1]
+    south = data.attrs['coord_lat_range'][0]
+
+    # 1/6th law to calculate the standard parallels. We calculate 1/6 of the
+    # distance in degrees from south to north, then add it to the southern
+    # figure limit and subtract from the northern limit.
+    #
+    law16 = (north - south) / 6
+    standard_parallels = (south + law16, north - law16)
+
+    # Central longitude and latitude are halfway from east to west
+    # and from south to north, respectively
+    # Central longitude
+    midway_lon = (east - west) / 2
+    central_longitude = east - midway_lon
+
+    # Central latitude
+    midway_lat = (north - south) / 2
+    central_latitude = north - midway_lat
+
+    lat_range = (south, north)
+    lon_range = (west, east)
+
+    proj_info = {'lat_range': lat_range,
+                 'lon_range': lon_range,
+                 'standard_parallels': standard_parallels,
+                 'central_longitude': central_longitude,
+                 'central_latitude': central_latitude,
+                 }
+
+    return proj_info
+
+
+def show_var_data_map(data, var, idepth, proj):
```

Comment:
Current version is crashing. Logic changed from using *kwargs* to args (i.e. `idepth=0, var=''`). Also need to change `proj` as *kwarg*